cli/tests/extensions/permissions.rs

```diff
@@ -114,3 +116,20 @@ pub async fn get_package_details() {
         .failure()
         .stderr("❗ Error: Requires net access to \"phylum.io\"\n");
 }
+
+#[test]
+fn permission_op() {
+    let test_cli = TestCli::builder().with_config(None).build();
+
+    let permissions =
+        Permissions { read: Permission::List(vec!["/tmp".to_string()]), ..Permissions::default() };
+
+    let permissions_ext = "
+         const perms = PhylumApi.permissions()
+         console.log(perms);";
+
+    let status =
+        test_cli.extension(permissions_ext).with_permissions(permissions).build().run().success();
+
+    println!("{}", String::from_utf8_lossy(&status.get_output().stdout));
```

Comment:
Why is there a `println` in this test? Shouldn't we assert the output?

Reply:
Missed that.